stag-library-compiler/src/test/java/com/vimeo/stag/processor/utils/DebugLogTest.kt

```diff
@@ -0,0 +1,30 @@
+package com.vimeo.stag.processor.utils
+
+import com.vimeo.stag.processor.StagProcessor
+import com.vimeo.stag.processor.Utils
+import org.junit.Test
+
+/**
+ * Unit tests for [DebugLog].
+ *
+ * Created by restainoa on 6/15/18.
+ */
+class DebugLogTest {
+
+    @Test
+    fun `DebugLog is not instantiable`() {
+        Utils.testZeroArgumentConstructorFinalClass(DebugLog::class.java)
+    }
+
+    @Test
+    fun `log message works`() {
+        StagProcessor.DEBUG = true
+        DebugLog.log("test message")
```

Comment:
This isn't really testing that logging is working. It's only testing that exceptions aren't being thrown when trying to log.

If it's worth actually testing this, then you could make `DebugLog` hold a `OutputStream` variable which defaults to `System.out` and mock an `OutputStream` during testing.

If you do this then also add some tests that test when `StagProcessor.DEBUG = false`.

Reply:
Made `DebugLog` take a logging implementation and then injecting that when the processor gets set up. It is now testable, good callout.